app/chatproxy/dbx_client.py

```
@@ -31,20 +30,20 @@ class DatabricksModel(BaseClient):
 
 	def __init__(self, 
 				 model_name: str = "databricks-meta-llama-3-3-70b-instruct",
-				 base_url: str = None):
+				 base_url: str = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)